src/app/interactions/drag-drop/email-sample/email-sample.component.scss

```diff
@@ -1,116 +1,92 @@
 @import "~igniteui-angular/lib/core/styles/themes/index";
 
-.mailbox {
-    -webkit-touch-callout: none; /* iOS Safari */
-    -webkit-user-select: none; /* Safari */
-     -khtml-user-select: none; /* Konqueror HTML */
-       -moz-user-select: none; /* Firefox */
-        -ms-user-select: none; /* Internet Explorer/Edge */
-            user-select: none; /* Non-prefixed version, currently
-                                  supported by Chrome and Opera */
+
+.drag-ghost {
+    position: relative;
     display: flex;
-    flex-direction: row;
-	font-weight: 400;
-    font-style: normal;
-    /* Clear default theming */
-    background: var(--igx-banner-banner-background);
+    align-items: center;
+    justify-content: flex-start;
+    width: rem(180px);
+    color: var(--igx-secondary-200-contrast);
+    font-size: rem(13px);
+    padding: rem(8px);
+    border-radius: rem(4px);
+    /* IE fallback */
+    background-color: #e41c77;
+    /* Default color */
+    background: var(--igx-secondary-200);
+
+    igx-icon {
+        margin-right: rem(8px);
+    }
 }
-.flexlist {
+
+:host {
     display: flex;
-    flex-direction: column;
-}
-.igx-list__item-content {
-    padding: 0.1rem 1rem;
-}
-.folders {
-    igx-list-item {
-        letter-spacing: 0.15px;
-        width: 13rem;
-        height: 3rem;
-        overflow: hidden;
+    user-select: none;
+	font-weight: 400;
+    font-style: normal;
+    height: 100%;
+
+    .folders {
+        width: rem(200px);
+        padding: 8px;
         /* IE fallback color */
-        background: igx-color($default-palette, 'grays', 200);
+        background: igx-color($default-palette, 'grays', 50);
         /* Default color */
-        background: var(--igx-grays-200);
-        cursor: pointer;
-        width: 13rem;
-        &:hover {
-            /* IE fallback color */
-            background: igx-color($default-palette, 'grays', 100);
-            /* Default color */
-            background: var(--igx-grays-100);
-        }
-        &:active {
-            /* IE Fallback color */
-            background: igx-color($default-palette, 'grays', 50);
-            /* Default color */
-            background: var(--igx-grays-50);
-        }
+        background: var(--igx-grays-50);
+        border: none;
+
```

Comment:
The hover color when dragging an email into the categories list is lost

Reply:
This should be fixed with the latest commit. Thanks! :)